src/main/java/edu/jhuapl/dorset/demos/DatabaseManager.java

```diff
@@ -0,0 +1,307 @@
+        return session;
+    }
+
+    /**
+     * Commit and close current Session
+     *
+     * @param session   Session to close
+     */
+    private void endSession(Session session) {
+        session.getTransaction().commit();
+        session.close();
+    }
+
+    /**
+     * Create a new item with given task
+     *
+     * @param task   the task for the new item
+     * @return todoItem   the new item to be added
+     */
+    private Item createItem(String task) {
+        Item todoItem = new Item();
+
+        int listNumber = getItemCount() + 1;
+        todoItem.setListNumber(listNumber);
+        todoItem.setTask(task);
+        todoItem.setDateCreated(getDate());
+        todoItem.setTimeCreated(getTime());
+
+        return todoItem;
+    }
+
+    /**
+     * get the number of items in the database
+     *
+     * @return itemCount   the number of items in the database
+     */
+    private int getItemCount() {
+        Session session = getSession();
+
+        String hql = "SELECT COUNT(item_id) FROM " + Item.class.getName();
+        Query query = session.createQuery(hql);
+        String response = getQueryResponse(query);
+        int itemCount = Integer.parseInt(response);
+        
+        endSession(session);
+        return itemCount;
+    }
+
+    /**
+     * Get response from query
+     *
+     * @param session   the current Session
+     * @param hql   the HQL query
+     * @return response   the query response
+     */
+    private String getQueryResponse(Query query) {
+        String response = query.list().toString();
+        response = response.substring(1, response.indexOf("]"));
+        return response;
+    }
+
+    /**
+     * Get the current date
+     * Formatted as: MM/dd/yyyy
+     *
+     * @return the current date
+     */
+    private String getDate() {
+        Date now = new Date();
+        DateFormat day = new SimpleDateFormat("MM/dd/yyyy");
+        return day.format(now);
+    }
+
+    /**
+     * Get the current time
+     * Formatted as: hh:mm AM/PM
+     *
+     * @return the current time
+     */
+    private String getTime() {
+        Date now = new Date();
+        DateFormat time = DateFormat.getTimeInstance(DateFormat.SHORT);
+        return time.format(now);
+    }
+
+    /**
+     * Remove an item from the database based on its item number
+     * 
+     * @return item.toString()   the string value of the item retrieved from the database
+     */
+    public String removeItem(int itemNumber) {
+        Session session = getSession();
+
+        Item item = (Item) session.createCriteria(Item.class)
+                        .add(Restrictions.eq("listNumber", itemNumber)).uniqueResult();
+
+        if (item == null) {
+            return "";
+        }
+        
+        deleteItemAndUpdateList(session, item);
+        endSession(session);
+        return item.toString();
+    }
+    
+    /**
+     * Remove an item from the database
+     *
+     * @return item removed
+     */
```

Comment:
It looks like you are returning an empty string rather than the item. I'd either return the item or return a boolean that indicates whether the item was removed.